packages/frontend/src/components/projects/sections/permissions/participants.tsx

```diff
@@ -21,20 +16,9 @@ export function ParticipantsEntry({
       {participants.map((address, i) => (
         <HighlightableLink
           key={i}
-          variant={
-            address.verificationStatus === 'unverified' ? 'danger' : undefined
-          }
           href={address.href}
           className="flex items-center gap-0.5"
         >
-          {address.verificationStatus === 'unverified' ? (
```

Comment:
Not needed anymore?

Reply:
Talked with basti, he said that showing if a participant is unverified doesn't make sense